### PR TITLE
CV - Whats supported

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/verify/cv-whats-supported.md [hunk 1]
```diff
@@ -5,51 +5,56 @@ sidebar_label: What's supported
...
 - Dynatrace
 - Google Cloud Operations (formerly Stackdriver)
 - New Relic
-- Prometheus
+- Prometheus (includes support for AWS Prometheus)
+- SignalFx
 - Splunk Observability
 - Sumologic
 
+Harness supports most of the popular APM tools, but there may be an instance where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such APM tools with Harness.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/verify/cv-whats-supported.md:31]:
```suggestion
Harness supports most of the popular APM tools, but there may be instances where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such APM tools with Harness.
```

File: docs/continuous-delivery/verify/cv-whats-supported.md [hunk 2]
```diff
@@ -5,51 +5,56 @@ sidebar_label: What's supported
...
 - Datadog
-- ELK
+- Elasticsearch
 - Google Cloud Operations (formerly Stackdriver)
 - Grafana Loki
-- Splunk
+- Splunk Observability
 - Sumologic
 
+Harness supports most of the popular log management tools, but there may be an instance where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such log management tools with Harness.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/verify/cv-whats-supported.md:45]:
```suggestion
Harness supports most of the popular log management tools, but there may be instances where Harness don't have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such log management tools with Harness.
```

**Review by roshnisarangadharan — APPROVED** [review]
Approved with one minor suggestion.

**Comment by SudheendraKatte** [comment]
@ravilach, please review and provide comments.

**Comment by ravilach** [comment]
@SudheendraKatte can you add here also? https://64b16cd98e0e0a363f784700--harness-developer.netlify.app/docs/getting-started/supported-platforms-and-technologies/#log-providers

**Pull request merged**